test/elixir/test/lots_of_docs_test.exs

```diff
@@ -0,0 +1,56 @@
+defmodule LotsOfDocsTest do
+    use CouchTestCase
+    
+    @moduletag :lots_of_docs
+    @docs_range 0..499
+
+    @moduledoc """
+    Test saving a semi-large quanitity of documents and do some view queries.
+    This is a port of the lots_of_docs.js suite
+    """
+
+    @tag :with_db
+    test "lots of docs", context do
+        db_name = context[:db_name]
+        @docs_range
+            |> create_docs()
+            |> Enum.chunk_every(100)
+            |> Enum.each(fn(docs) -> bulk_post(docs, db_name) end)
+
+        %{"rows" => rows, "total_rows" => total_rows} = Couch.get("/#{db_name}/_all_docs").body
```

Comment:
The javascript test seems to create a map/reduce view before querying it. Here is looks like  you using `_all_docs`

Reply:
Testing `_all_docs` is cool. But I think lets test a view as well.

Could you create another test that runs again a view.

Reply:
@garrensmith cool. I think I addressed this with my latest fixup, could you have a look? I created a whole new test for this purpose.